pages/community.vue

```diff
@@ -32,10 +32,10 @@ definePageMeta({
           </Subhead>
           <div class="mx-auto flex justify-between gap-8">
             <div>
-              <AppLink href="https://filecoin.io/slack">
-                <img src="~/assets/images/circle-slack.svg" class="mx-auto my-4 block">
+              <AppLink href="https://matrix.to/#/#ipfs-space:ipfs.io">
```

Comment:
@TheDiscordian  any concerns around linking to `#ipfs-space` (space), instead of `#lobby` channel?